### ISSUE TITLE
Additional of useful jupyterlab extensions

### ISSUE THREAD

**Issue by costrouc** [nebari-dev/nebari-docker-images#53]
### Context

Jupyterlab has a rich list of extensions

 - 

### Value and/or benefit

asdf

**Comment by viniciusdc** [comment]
@dharhas was this the S3 browser extension that you commented? https://github.com/IBM/jupyterlab-s3-browser

**Comment by dharhas** [comment]
Yes. But I've never used it. More of a suggestion of something that might be useful.

**Comment by viniciusdc** [comment]
As we discoussed recently in our internal sync:
## Accepted
- [spellchecker](https://jupyterlab-contrib.github.io/spellchecker.html): Spellchecker for JupyterLab notebook markdown cells and file editor.
- [jupyterlab-execute-time](https://github.com/deshaw/jupyterlab-execute-time): Execute Time Plugin for Jupyter Lab
## Pending acceptance 
- [jupyterlab-spreadsheet-editor](https://jupyterlab-contrib.github.io/jupyterlab-spreadsheet-editor.html): JupyterLab spreadsheet editor for tabular data (e.g. csv, tsv)
- [jupyterlab-sql](https://github.com/pbugnion/jupyterlab-sql): SQL GUI for JupyterLab

**Comment by krassowski** [comment]
>  https://github.com/jtpio/jupyterlab-system-monitor

this was merged into  https://github.com/jupyter-server/jupyter-resource-usage (along with kernel usage monitor that we contributed) which is already included:

https://github.com/nebari-dev/nebari-docker-images/blob/f941abcf688eef6708dda10d91b3ad3f69991ce7/jupyterlab/environment.yaml#L32

**Comment by krassowski** [comment]
I had started porting spreadsheet editor to Lab 4.0 in my free time some time ago, let me know if this would be of interest.

**Comment by pavithraes** [comment]
> I had started porting spreadsheet editor to Lab 4.0 in my free time some time ago, let me know if this would be of interest.

Personally, I think can be quite useful. :)

**Comment by krassowski** [comment]
[jupyterlab-spreadsheet-editor](https://github.com/jupyterlab-contrib/jupyterlab-spreadsheet-editor) `v0.7.1` now supports JupyterLab 4+. As author I am biased, but I believe this has a lot of potential, especially after some UI review (the current toolbar is ugly IMO) and after addition of more engines to enable reading `.xlsx` and `.ods` files.
